backend/inspirehep/records/api/literature.py

```diff
@@ -317,6 +318,31 @@ def add_figures(self, figures):
     def add_file(
         self, url, original_url=None, key=None, filename=None, *args, **kwargs
     ):
+        if current_s3_instance.is_s3_url(url) and not current_app.config.get(
+            "FORCE_PROCESS_S3_FILES", False
+        ):
+            result = {
+                "key": key,
+                "filename": filename,
+                "url": url,
+                "original_url": original_url,
+                **kwargs,
+            }
+            LOGGER.info(
+                "File already on S3 - Skipping",
+                url=url,
+                key=key,
+                recid=self.get("control_number"),
+                uuid=self.id,
+            )
+            return strip_empty_values(result)
+        LOGGER.info(
```

Comment:
i don't think this log is necessary as we have the other 2 with replacing file metadata and adding file to s3 already

Reply:
:+1: